ibm_catalog.json

```diff
@@ -63,6 +63,174 @@
                 "key": "existing_cos_kms_key_crn",
                 "required": true
               },
+              {
+                "key": "log_analysis_plan",
+                "required": true,
+                "default_value": "7-day",
+                "options": [
+                  {
+                    "displayname": "lite",
+                    "value": "lite"
+                  },
+                  {
+                    "displayname": "7-day",
+                    "value": "7-day"
+                  },
+                  {
+                    "displayname": "14-day",
+                    "value": "14-day"
+                  },
+                  {
+                    "displayname": "30-day",
+                    "value": "30-day"
+                  },
+                  {
+                    "displayname": "hipaa-30-day",
+                    "value": "hipaa-30-day"
+                  }
+                ]
+              },
+              {
+                "key": "log_analysis_service_endpoints",
+                "required": true,
+                "default_value": "private",
+                "options": [
+                  {
+                    "displayname": "public",
+                    "value": "public"
+                  },
+                  {
+                    "displayname": "private",
+                    "value": "private"
+                  },
+                  {
+                    "displayname": "public-and-private",
+                    "value": "public-and-private"
+                  }
+                ]
+              },
+              {
+                "key": "cloud_monitoring_plan",
+                "required": true,
+                "default_value": "graduated-tier",
```

Comment:
you can remove `default_value` here so we don't have to maintain it in 2 places. If not specified here, it will pick the default from terraform.